scripts/buildkite/rebuild.hs

```diff
@@ -13,7 +13,7 @@ import Turtle
 -- | Run build and upload coverage information when successful
 main :: IO ()
 main = do
-  buildResult <- buildStep (Just ["--scenario=ContinuousIntegration"])
+  buildResult <- buildStep Nothing
```

Comment:
what is the result of this change ?

Reply:
Just removes a test flag that's not used in this package